docs/hardware.rst

```diff
@@ -391,33 +422,24 @@ We have tested and can recommend the `NUC8i5BEK <https://www.intel.com/content/w
 It provides a single storage option: an M.2 NVMe or SATA SSD.
 
 The NUC8i5BEK has soldered-on wireless components, which cannot easily be
-removed. For security reasons, we recommend that you take the following steps
-to disable wireless functionality:
-
-- before installation of the RAM and storage, disconnect the wireless antennae
-  leads:
+removed. Before installation of the RAM and storage, we recommend that you
+disconnect the wireless antennae leads:
 
 |NUC8 leads|
 
-- before the initial OS installation, boot into the BIOS by pressing **F2** at
-  startup, navigate to **Advanced > Devices > Onboard Devices**, and disable
-  unwanted hardware - everything except **LAN**:
-
-|NUC8 VisualBios1|
-
-- navigate to **Advanced > Security** in the BIOS and disable SGX support, which is not used by 
-  SecureDrop and may be targeted by active CPU exploits:
+Before the initial OS installation, boot into the BIOS by pressing **F2** at
+startup and adjust the system configuration:
 
-|NUC8 VisualBios2|
+- Under **Advanced ▸ Devices ▸ Onboard Devices**, disable all onboard devices
+  other than LAN: audio, audio DSP, microphone, Thunderbolt, WLAN, Bluetooth,
+  and SD card.
 
-- navigate to **Advanced > Boot > Secure Boot** and uncheck the **Secure Boot** checkbox:
+- Under **Advanced ▸ Security**, disable SGX support, which is not used by
+  SecureDrop and may be targeted by active CPU exploits.
 
-|NUC8 VisualBIOS SecureBoot|
+- Under **Advanced ▸ Boot ▸ Secure Boot**, uncheck the **Secure Boot** checkbox.
 
 .. |NUC8 leads| image:: images/hardware/nuc8_leads.jpg
-.. |NUC8 VisualBIOS1| image:: images/hardware/nuc8_visualbios1.png
-.. |NUC8 VisualBIOS2| image:: images/hardware/nuc8_visualbios2.png
-.. |NUC8 VisualBIOS SecureBoot| image:: images/hardware/nuc8_visualbios_secureboot.png
```

Comment:
As noted in the PR summary, I'm removing these screenshots to increase maintainability of the docs, given that the menu structure is generally straightforward to navigate.